packages/toucan-js/src/sdk.ts

```diff
@@ -91,4 +100,40 @@ export class Toucan extends Hub {
     const client = this.getClient<ToucanClient>() as ToucanClient;
     return client.captureCheckIn(checkIn, monitorConfig, scope);
   }
+
+  /**
+   * Add a breadcrumb to the current scope.
+   */
+  addBreadcrumb(breadcrumb: Breadcrumb, maxBreadcrumbs: number = 100): this {
+    const client = this.getClient<ToucanClient>() as ToucanClient;
+    const max = client.getOptions().maxBreadcrumbs || maxBreadcrumbs;
+
+    return super.addBreadcrumb(breadcrumb, max);
+  }
+
+  /**
+   * Creates a new scope with and executes the given operation within.
+   * The scope is automatically removed once the operation
+   * finishes or throws.
+   */
+  withScope<T>(callback: (scope: Toucan) => T): T {
```

Comment:
If we export the globals like `withScope` wouldn't we then need to store the client somewhere globally too?

I wanted to avoid any global clients like toucan was for v7.

Currently `client.withScope` clones the client (which is tricky because `Scope` was never designed to be extended) so that when the scope is modified it only applies to this new client. It also means if Sentry add methods to `Scope`, toucan gets these automatically.

It might be possible to do this in a cleaner way with a PR at `@sentry/core`